app/src/main/java/com/github/swent/echo/data/model/Event.kt

```diff
@@ -9,10 +9,14 @@ data class Event(
     val eventId: String,
     val creatorId: String,
     val organizerId: String,
+    val organizerName: String,
     val title: String,
     val description: String,
     val location: Location,
-    @Contextual val startDate: Date,
-    @Contextual val endDate: Date,
-    val tags: Set<Tag>
+    @Contextual val startDate: ZonedDateTime,
+    @Contextual val endDate: ZonedDateTime,
+    val tags: Set<Tag>,
+    val participantCount: Int,
```

Comment:
not sure about the participantCount, to me it would make more sense that this is linked to a SQL request that views the relationship table between an event and a user (meaning that the user is enrolled in the event) and count on it rather than have a count as a field, for the rest of the changes to event it's good 👍

Reply:
From what I understand, the SQL query to get the participant count would be the responsibility of whoever builds the `Event` object, in this case the repository, but I'm not sure.

@violoncelloCH can you confirm?

Reply:
yes exactly! 
sorry I did not have the time to review this this morning :D